tests/dm_syncer/conf/dm-syncer-2.toml

```diff
@@ -0,0 +1,124 @@
+# Base Configuration.
```

Comment:
how about this config file use old config format?

Reply:
ok